src/bindings/interface/units.i

```diff
@@ -65,6 +65,10 @@ Only the first matching unit is removed.
 "Returns the attributes of the unit specified by index, reference, or
 StandardUnit.";
 
+%feature("docstring") libcellml::Units::requiresImports
+"Returns 'True' when this Units object depends on an imported child
+Units object, or 'False' otherwise.";
```

Comment:
Minor: replace `'`' with `` ` ``. This is what (most) other interfaces use.

Reply:
We are super inconsistent throughout ... there are lots of back ticks and single string quotes everywhere!

Reply:
So we should add it in to the style guide doc that we want back ticks so we know which style we should use.

Reply:
Done (for this instance).